src/Manager/Flexible.php

```diff
@@ -115,18 +120,20 @@ protected function spawnAndGetMessenger(callable $current)
 
     public function ping()
     {
+        if (count($this->workers) + $this->startingProcesses < $this->options[Options::MIN_SIZE]) {
+            for ($i = count($this->workers) + $this->startingProcesses; $i <= $this->options[Options::MIN_SIZE]; $++) {
```

Comment:
Tiny parse error: https://travis-ci.com/WyriHaximus/reactphp-child-process-pool/jobs/250552303#L416

```suggestion
            for ($i = count($this->workers) + $this->startingProcesses; $i <= $this->options[Options::MIN_SIZE]; $i++) {
```